index_codec.md

```diff
@@ -33,23 +33,23 @@
 
 .# Abstract
 
-This document defines the Matroska codec mappings, including the codec ID, layout of data in a `Block` and in an optinal `CodecPrivate`.
+This document defines the Matroska codec mappings, including the codec ID, layout of data in a `Block Element` and in an optional `CodecPrivate Element`.
 
 {mainmatter}
 
 # Introduction
 
-Matroska aims to become THE standard of multimedia container formats. It stores interleaved and timestamped audio/video/subtitle data using various codec. To interpret the codec data, a mapping between the way the data are stored in Matroska and how they are understood by such codec is necessary.
+Matroska aims to become THE standard of multimedia container formats. It stores interleaved and timestamped audio/video/subtitle data using various codecs. To interpret the codec data, a mapping between the way the data is stored in Matroska and how it is understood by such a codec is necessary.
 
-This document intends to define this mapping for many commonly used codec in Matroska.
+This document intends to define this mapping for many commonly used codecs in Matroska.
 
 # Status of this document
 
 This document is a work-in-progress specification defining the Matroska file format as part of the [IETF Cellar working group](https://datatracker.ietf.org/wg/cellar/charter/). It uses basic elements and concept already defined in the Matroska specifications defined by this workgroup.
 
 # Security Considerations
 
-Matroska Codecs inherits security considerations from EBML and Matroska.
+Matroska Codecs inherit security considerations from EBML and Matroska.
```

Comment:
I just realized that "Matroska Codecs" is supposed to refer to the title of this document. Therefore "inherits" would be correct. Indeed, the title attribute at the very top is "Matroska Codec" (singular), something that I'd like to see changed to the plural.

As to this line (and similar lines in the other two documents): maybe changing it to "This document inherits… from the documents…" would make it sound less confusing.